pkg/common/consumer/consumer_handler.go

```diff
@@ -72,6 +83,7 @@ func NewConsumerHandler(logger *zap.SugaredLogger, handler KafkaConsumerHandler,
 	sch := SaramaConsumerHandler{
 		handler:           handler,
 		lifecycleListener: noopSaramaConsumerLifecycleListener{},
+		timeout:           60 * time.Second, // default rebalance timeout
```

Comment:
default... - it's more hard coded, atm - right?

Reply:
can be overridden by composing the factory function `NewConsumerHandler` with `WithTimeout` option .